collector/eventhub/src/main/java/zipkin/collector/eventhub/IEventProcessorHostFactory.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2017 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin.collector.eventhub;
+
+public interface IEventProcessorHostFactory {
```

Comment:
This is simply to wrap the monolithic implementation to allow testing. Then again, the method interface is in the azure library, I am just mimicking it.